src/NLog/Common/InternalLogger.cs

```diff
@@ -367,6 +344,64 @@ private static bool LoggingEnabled(LogLevel logLevel)
                    LogWriter != null;
         }
 
+        private static void WriteToLogFile(string message)
```

Comment:
Please add some docs, most important is here that logging will be skipped if not enabled. (multiple times)

Reply:
I will do